recipes/lunarcalendar/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Can you open a PR upstream to add `LICENCE` to `MANIFEST.in`. We don't need to wait for it to be released, it's just better so conda-forge can't get out of sync if it ever changes. (And technically the PyPI tarball is in violation of the licence without it.)

Reply:
Done in https://github.com/wolfhong/LunarCalendar/pull/8